.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,11 @@
+Checklist for Pull Requests
 - [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
+- [ ] Is there a linked Issue
```

Comment:
YamlCreate will handle this fine, but it should be updated to handle it a bit more seamlessly

Reply:
This has been resolved with the latest pushes